infra/main.bicep

```diff
@@ -42,6 +42,7 @@ param appServiceSkuName string // Set in main.parameters.json
 param openAiHost string // Set in main.parameters.json
 param isAzureOpenAiHost bool = startsWith(openAiHost, 'azure')
 param azureOpenAiCustomUrl string = ''
+param azureOpenAiApiVersion string = ''
```

Comment:
Could you also add to outputs at the bottom, just in case someone sets it by changing line 45? (vs using `azd env set`)